src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.contexts.wallet.events.collectibles
```

Comment:
moved the collectibles file to an events.cljs file within it's collectible folder to align with the other events file